app/helpers/application_helper.rb

```diff
@@ -1,2 +1,21 @@
 module ApplicationHelper
+	def logged_in?
+    session[:user_id] != nil
+  end
+
+  def current_user
```

Comment:
could we change this to ~~~~ @cached user ||= User.find(session[:user_id]) ~~~~ which would save a db request for every new page load...I'm going to merge anyway just a thought for refactoring...